### DISCUSSION TITLE
SvelteKit build stuck on 'built in X.XXs'

### DISCUSSION THREAD

**Discussion by lolzivkovic** [vercel/community#2697] [Help]
### Summary

Recently my builds are getting stuck being built on message 'built in 9.05s' for example.
I tried using pnpm and npm and same thing happens. Here is my current project package.json
`{
	"name": "example",
	"version": "0.0.1",
	"private": true,
	"scripts": {
		"preinstall": "npx only-allow pnpm",
		"dev": "vite dev --host",
		"build": "vite build",
		"preview": "vite preview",
		"lint": "prettier --check . && eslint .",
		"format": "prettier --write ."
	},
	"devDependencies": {
		"@sveltejs/adapter-vercel": "^3.0.0",
		"@sveltejs/kit": "^1.20.0",
		"autoprefixer": "^10.4.14",
		"eslint": "^8.41.0",
		"eslint-config-prettier": "^8.8.0",
		"eslint-plugin-svelte3": "^4.0.0",
		"postcss": "^8.4.24",
		"prettier": "^2.8.8",
		"prettier-plugin-svelte": "^2.10.1",
		"svelte": "^3.59.1",
		"svelte-preprocess": "^5.0.4",
		"tailwindcss": "^3.3.2",
		"vite": "^4.3.9"
	},
	"type": "module",
	"dependencies": {
		"@sendgrid/mail": "^7.7.0",
		"mongodb": "^5.5.0",
		"svelte-email": "^0.0.4",
		"swiper": "^9.3.2"
	}
}
`
and here is my svelte.config.js 
`import adapter from '@sveltejs/adapter-vercel';
import preprocess from 'svelte-preprocess';

/** @type {import('@sveltejs/kit').Config} */
const config = {
	preprocess: [preprocess({ postcss: true })],
	kit: {
		paths: {
			base: ''
		},
		adapter: adapter(),
	}
}

export default config`

### Steps to Reproduce

One of the development builds when reverting back to pnpm worked out once and the same build cannot be promoted to production. It gets stuck on 'built in' message

**Comment by swarnava** [comment]
Hi, do you have a deployment ID / Link?

**Reply by lolzivkovic** [reply]
@swarnava Thanks for replying,

it seems it was a problem with MongoDB 5.5.0. I reverted back to 5.4.0 version and after redeploying without cache it went through successfully. 

Here is the [Link](https://force-luxury-cars-2vc788mnf-junitef.vercel.app) of the deployment that I canceled after 23 minutes.
